validator/tests/test_validator.py

```diff
@@ -1,14 +1,45 @@
-""""""
-from datetime import date, datetime
-
-from delphi_validator.datafetcher import FILENAME_REGEX, make_date_filter
+"""Tests for Validator"""
+import pytest
 from delphi_validator.validate import Validator
 
 class TestValidatorInitialization:
-
+    """Tests for proper initialization."""
     def test_default_settings(self):
+        """Test default initialization."""
         params = {"data_source": "", "span_length": 0,
                   "end_date": "2020-09-01", "expected_lag": {}}
         validator = Validator(params)
         assert len(validator.suppressed_errors) == 0
         assert isinstance(validator.suppressed_errors, set)
+
+    def test_suppressed_errors(self):
+        """Test initialization with suppressed errors."""
+        params = {"data_source": "",
+                  "span_length": 0,
+                  "end_date": "2020-09-01",
+                  "expected_lag": {},
+                  "suppressed_errors": [["a", "b"], ["c", "d"], ["a", "b"]]
+                 }
+
+        validator = Validator(params)
+        assert not validator.suppressed_errors ^ set([("a", "b"), ("c", "d")])
```

Comment:
Is this just checking for equality? If so, I believe `==` will work and be more readable.

Reply:
Yes, done